app/utils/tx.ts

```diff
@@ -448,7 +448,7 @@ export const PROGRAM_INFO_BY_ID: { [address: string]: ProgramInfo } = {
     },
 };
 
-export type LoaderName = typeof LOADER_IDS[keyof typeof LOADER_IDS];
+export type LoaderName = (typeof LOADER_IDS)[keyof typeof LOADER_IDS];
```

Comment:
Nit: Can you remove the parantheses here?

Reply:
I think this is actually a new [Prettier feature](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEcAeAHCAnGACGATwzjwBkIBDAEzmwDlKBbUgXgOLggDNyB5AIIARAKIAlAPoBJIQGUA2gGs4hHhxJqyg0ZJmyAugG4QAGhAQMMAJbQAzslCVs2CAHcACk4T2UlADaulIT2ZgBG2JRgyjCyzHBkVlBwyNz+tnBhEVFwMRiRiQDmyDDYAK4ZqEyhcNS01GSUUAWllAVwAGI4TJQw1k3IIJSlMBCmIAAWMEx+AOrjVvC2eWBwst4LVgBuC4QDYLYhIInpuO4RBd0paRUAVrZosoV+cACKpRDwV37pZnnYJwMiCRbGBsFZLGMMGDYDMrNQYONkAAOAAMvxc6RmEQwAyhcBOm2SZgAju94GcLD5BrYALRJGo1MbYOCkqzMs6tS5IVLfCrpJhWYplPlPOACXpg0LDODuOgJJJfH4gWyit4fZLc65mGCUUKw+GIpAAJm1ESsfkKAGEIEwuahbABWMaldIAFV1Ph5Ss25SkUFosFkoPBMAE-tkRGeirgAF8Y0A). We should upgrade Prettier to the latest and make these changes.